File: examples/sakila/python/04_has_many_through.py
from __future__ import annotations

import json

from common import load_db


def main() -> None:
    db = load_db()
    query = {
        "films": {
            "pk": "714",
            "include": {
                "actors": {"fields": ["actor_id", "first_name", "last_name"]}
            },
        }
    }
    result = db.execute(query)
    print(json.dumps(result, indent=2))


if __name__ == "__main__":
    main()

File: examples/sakila/rust/src/bin/04_has_many_through.rs
use serde_json::json;
use virtuus_sakila_examples::load_db;

fn main() {
    let mut db = load_db();
    let query = json!({
        "films": {
            "pk": "714",
            "include": {
                "actors": {"fields": ["actor_id", "first_name", "last_name"]}
            }
        }
    });
    let result = db.execute(&query);
    println!("{}", serde_json::to_string_pretty(&result).unwrap());
}
